<ui version="4.0" >
 <class>CSwissRangerParamsGuiComp</class>
 <widget class="QWidget" name="CSwissRangerParamsGuiComp" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>130</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Modulation Frequency</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Amplitude Threshold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Distance Clipping</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QComboBox" name="ModulationFrequencyModeCombo" >
         <item>
          <property name="text" >
           <string>40MHz</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>30MHz</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>21MHz</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>20MHz</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>19MHz</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>60MHz</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>15MHz</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>10MHz</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>29MHz</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>31MHz</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="AmplitudeThresholdSpin" >
         <property name="suffix" >
          <string>%</string>
         </property>
         <property name="decimals" >
          <number>2</number>
         </property>
         <property name="maximum" >
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.010000000000000</double>
         </property>
         <property name="value" >
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>From</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="DistanceClippingFromSpin" >
           <property name="suffix" >
            <string>%</string>
           </property>
           <property name="decimals" >
            <number>2</number>
           </property>
           <property name="maximum" >
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>To</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="DistanceClippingToSpin" >
           <property name="suffix" >
            <string>%</string>
           </property>
           <property name="decimals" >
            <number>2</number>
           </property>
           <property name="maximum" >
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QCheckBox" name="MedianFilterCheck" >
       <property name="text" >
        <string>Enable Median Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="AdaptiveFilterCheck" >
       <property name="text" >
        <string>Enable Adaptive Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
